.circleci/config.yml

```diff
@@ -373,7 +373,7 @@ workflows:
           filters:
             branches:
               ignore:
-                - develop
+                - trunk
                 - /^release.*/
                 - /^gutenberg\/integrate_release_.*/
                 - /pull\/[0-9]+/
```

Comment:
💄  Nit: Unrelated to this PR but I'm pretty sure this `/pull\/[0-9]+/` branch regex is obsolete? (And removing it would allow is to use a YAML anchor to mutualize the list with the one on lines 360 above, reducing maintenance).
Could totally be kept for a later PR (maybe when we'll migrate this to Buildkite) though.

Reply:
Looks like it's obsolete, but I am not a fan of mixing changes in PRs. I don't think it's worth making that change right now since we'll migrate it to Buildkite very soon, but even if I did, I wouldn't do it in this PR as it's completely unrelated.

Reply:
Yep, totally with you on this 👍